### ISSUE TITLE
Link to starter docs

### ISSUE THREAD

**Issue by sdelamo** [micronaut-projects/static-website#164]
We link to micronaut-starter docs. We probably need to add micronau-starter to modules.yml

**Comment by JasonTypesCodes** [comment]
@sdelamo is this still relevant?

**Comment by sdelamo** [comment]
Yes, we don't have a link to [starter docs](https://micronaut-projects.github.io/micronaut-starter/latest/guide/) in the [new website documentation page](https://micronaut.io/docs/).
